packages/webdriverio/src/commands/element/scrollIntoView.ts

```diff
@@ -104,7 +104,7 @@ export async function scrollIntoView (
         deltaY = Math.round(deltaY - scrollY)
 
         await browser.action('wheel')
-            .scroll({ duration: 0, x: deltaX, deltaY, origin: this })
+            .scroll({ duration: 0, deltaX, deltaY, origin: this })
```

Comment:
This should be x, what is wrong here is that y was not added. To this should be y: deltaY

Reply:
Ok I'll check that. Thanks.

Reply:
You can run "npm i <path to package on disk>" and it will use the local changes from the webdriverio repo

Reply:
I still have trouble to test in an external repo, because of dependences issue, but I updated my commit as you asked. You can re-review.
Thanks